### PR TITLE
Kubectl boolean condition clean up 

### PR DESCRIPTION
**Pull request by weikhor**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
To clean up false/true condition in kubectl section
#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note

```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: staging/src/k8s.io/kubectl/pkg/cmd/set/set_subject_test.go [hunk 1]
```diff
@@ -412,7 +412,7 @@ func TestAddSubject(t *testing.T) {
 	for _, tt := range tests {
 		changed := false
 		var got []rbacv1.Subject
-		if changed, got = addSubjects(tt.existing, tt.subjects); (changed != false) != tt.wantChange {
+		if changed, got = addSubjects(tt.existing, tt.subjects); (changed) != tt.wantChange {
```
Comment (brianpursley) [inline staging/src/k8s.io/kubectl/pkg/cmd/set/set_subject_test.go:415]: I guess it you might as well change it from `(changed) != tt.wantChange` to `changed != tt.wantChange`, right?

**Review by brianpursley — CHANGES_REQUESTED** [review]

File: staging/src/k8s.io/kubectl/pkg/cmd/set/set_subject_test.go (hunk 1 above)
Comment (weikhor) [reply staging/src/k8s.io/kubectl/pkg/cmd/set/set_subject_test.go]: Yes. Have updated

**Review by soltysh — CHANGES_REQUESTED** [review]
I'd suggest combining these kind of changes with some other refactorings.

**Comment by SergeyKanzhelev** [comment]
/close

this falls under the trivial edits policy and doesn't bring much value to codebase. Please consider finding issues with help wanted tag to contribute

**Pull request closed without merging**